src/poset/poset.go

```diff
@@ -1593,39 +1590,46 @@ func (p *Poset) Reset(block Block, frame Frame) error {
 //method call, the Poset should be in a state coherent with the 'tip' of the
 //Poset
 func (p *Poset) Bootstrap() error {
-	if badgerStore, ok := p.Store.(*BadgerStore); ok {
-		//Retreive the Events from the underlying DB. They come out in topological
-		//order
-		topologicalEvents, err := badgerStore.dbTopologicalEvents()
-		if err != nil {
-			return err
-		}
+	badgerStore, ok := p.Store.(*BadgerStore)
+	if !ok {
+		return nil
+	}
+	//Retreive the Events from the underlying DB. They come out in topological
+	//order
+	topologicalEvents, err := badgerStore.dbTopologicalEvents()
```

Comment:
This will not work on big databases which do not fit into memory, as dbTopologicalEvents() returns all events in the database.

Reply:
I confirm this, but I propose to solve this problem later. The application should work at least on small bases.

Reply:
Why this change makes it work?

Reply:
This change does not make the application work on large databases. But it will work on small bases.